site/pages/our-work/case-study/what-to-read-next/index.js

```diff
@@ -1,64 +1,57 @@
+// @flow
+import _ from 'lodash';
 import React from 'react';
-import classnames from 'classnames/bind';
-import InlineSVG from 'svg-inline-react';
+
 import styles from './style.css';
 import Link from '../../../../components/link';
+import WhatToReadNextSlice from './slice';
 
 import fortnumImage from './images/fortnumImage.jpg';
 import financialTimesImage from './images/financialTimesImage.png';
 import camdenMarketImage from './images/camdenMarketImage.jpg';
-import arrowSVG from '../../../../../assets/images/SVG/arrow.svg';
+import retailerImage from './images/retailerImage.jpg';
+
+const slices = {
+  fortnumAndMason: {
+    name: 'Fortnum & Mason',
+    tagline: 'Elegant e-commerce in eight months',
+    image: fortnumImage,
+    link: '/our-work/case-study/fortnum-and-mason/',
+  },
+  financialTimes: {
+    name: 'Financial Times',
+    tagline: 'Lasting change for a media giant',
+    image: financialTimesImage,
+    link: '/our-work/case-study/financial-times/',
+  },
+  camdenMarket: {
+    name: 'Camden Market',
+    tagline: 'Taking steps towards a digital future',
+    image: camdenMarketImage,
+    link: 'our-work/case-study/camden-market/',
```

Comment:
@martinhartt - Camden Market and Retailer are missing a / from the start of their link.